test/Microsoft.AspNetCore.Mvc.Abstractions.Test/Microsoft.AspNetCore.Mvc.Abstractions.Test.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Testing" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />
     <PackageReference Include="Moq" Version="$(MoqVersion)" />
+    <PackageReference Include="System.Reflection.TypeExtensions" Version="$(CoreFxVersion)" />
```

Comment:
For some reason when I build Universe with everything net46, I didn't need this. But building Mvc when we have upstream dependencies cross-compiling for `net451` causes some weirdness. Other repos also had similar oddities. I've filed https://github.com/aspnet/Mvc/issues/6005 to remove this or at least figure out why we need this once I've updated everything in Universe